tests/scripts/validate_cluster.sh

```diff
@@ -62,8 +62,12 @@ function test_demo_osd {
 }
 
 function test_demo_rgw {
-  # shellcheck disable=SC2046
-  return $(wait_for_daemon "$EXEC_COMMAND -s | grep -sq 'rgw:'")
```

Comment:
Should we consider updating `wait_for_daemon` to check for `Running`? @travisn ?

Reply:
Lookks good now waiting for the pod to be ready

Reply:
`Status.Phase` is Running even when Pod is in terminating status and also even when before all the contianers are ready